custom/icds_reports/migrations/sql_templates/database_views/ccs_record_monthly_view.sql

```diff
@@ -67,7 +67,8 @@ CREATE VIEW ccs_record_monthly_view AS
         "ccs_record_monthly"."lactating" AS "lactating",
         "ccs_record_monthly"."dob" AS "dob",
         "ccs_record_monthly"."open_in_month" AS "open_in_month",
-        "ccs_record_monthly"."closed" AS "closed"
+        "ccs_record_monthly"."closed" AS "closed",
+        "ccs_record_monthly"."anc_abnormalities" AS "anc_abnormalities"
```

Comment:
Looks like this does a `DROP` then `CREATE`. Is there a reason this query doesn't use `CREATE OR REPLACE` instead, since you are just adding columns?

Reply:
Not sure, but given how infrequently these are rebuilt, I am not sure it matters, and it does give us the flexibility to make any change, without changing the top commands.